module/documents/activity/summon.mjs

```diff
@@ -347,7 +347,7 @@ export default class SummonActivity extends ActivityMixin(SummonActivityData) {
     actorUpdates["flags.dnd5e.summon"] = {
       level: this.relevantLevel,
       mod: rollData.mod,
-      origin: this.uuid,
+      origin: this.item.uuid,
```

Comment:
It would still be good to track the specific activity, so maybe just stick `activity: this.id` in here.

Reply:
Added in 85e6fc8